docs/sql-data-sources-text.md

```diff
@@ -38,3 +36,36 @@ Spark SQL provides `spark.read().text("file_name")` to read a file or directory
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of text can be set via:
+* the `.option`/`.options` methods of
+  *  `DataFrameReader`
+  *  `DataFrameWriter`
+  *  `DataStreamReader`
+  *  `DataStreamWriter`
+  *  `OPTIONS` clause at [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
```

Comment:
Sorry I missed it here. This is not in "the `.option`/`.options` methods of". Can you make the level down?

Reply:
Sure. let me fix it in https://github.com/apache/spark/pull/32658